ParkingLot/ParkingBoy.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.ConstrainedExecution;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Parking
+{
+    public class ParkingBoy
+    {
+        private ParkingLot parkingLot = new ParkingLot();
+        private ParkingLot parkingLot2 = new ParkingLot();
+        private int maxCapacity = 10;
+
+        public string Park(string car, string strategy)
```

Comment:
In strategy pattern, strategy may should not as formal param(I'm not sure), but it seems use a class not  a string, https://refactoringguru.cn/design-patterns/strategy/csharp/example